src/Docs.scss

```diff
@@ -0,0 +1,13 @@
+.docs {
+  .sidebar {
+    min-width: 100px;
+    max-width: 200px;
+    @media screen and (max-width: 800px) {
+      display: none;
```

Comment:
It would be good if the TOC was just at the top of the mobile webpage, rather than having a close/open action.